DEVELOPMENT.md

```diff
@@ -185,13 +185,14 @@ Then navigate to the Swagger: `http://localhost:8080/swagger-ui.html`
 
 New images are built and pushed automatically each time a PR is merged. Developers may
 want to do this manually in order to test in-development CBAS versions in non-local
-environments. 
+environments.
 
 First, log into GCP using `gcloud auth login`. You need this to authenticate to push the image.
+You may also need to run `gcloud auth configure-docker` prior to pushing the image.
```

Comment:
Ran into this issue while testing so wanted to add a tip here for future reference.